opm/autodiff/FlowMainEbos.hpp

```diff
@@ -700,6 +777,9 @@ namespace Opm
         Grid& grid()
         { return ebosSimulator_->gridManager().grid(); }
 
+        const Grid& globalGrid()
+        { return *globalGrid_; }
```

Comment:
I do not like bloating up CpGrid with even more non-DUNE-interface methods. In addition I do not want to promote this usage of CpGrid as it is only there as crook because there is currently no other way to make Eclipse IO work.